edge/Dockerfile

```diff
@@ -1,51 +1,79 @@
 # DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 #
-# Copyright (c) 2019 Axway Software SA and its affiliates. All rights reserved.
+# Copyright (c) 2020 Axway Software SA and its affiliates. All rights reserved.
 #
-FROM centos:7 as base_image
+FROM registry.access.redhat.com/ubi8/ubi:8.0 as base_image
+
+LABEL maintainer="Axway Software SA <https://support.axway.com>"
+LABEL org.opencontainers.image.vendor="Axway"
+LABEL org.opencontainers.image.documentation="https://docs.axway.com/category/mft"
+LABEL org.opencontainers.image.title="SecureTransport Edge"
+LABEL org.opencontainers.image.description="SecureTransport Edge Image"
+LABEL org.opencontainers.image.url="https://git.ecd.axway.int/cloud/docker-st.git"
+LABEL org.opencontainers.image.source="git@git.ecd.axway.int:cloud/docker-st.git"
+LABEL org.opencontainers.image.documentation="https://docs.axway.com/category/mft"
```

Comment:
1) use the github rep for the links (see my previous comment for the backend)
2) Remove the last _**org.opencontainers.image.documentation="https://docs.axway.com/category/mft"**_ as it's duplicate